app/views/components/gds/addButton.scala.html

```diff
@@ -20,15 +20,14 @@
 
 @this(govukButton : GovukButton)
 
-@(hiddenLabel: Option[String] = None)(implicit messages: Messages)
+@(label: String = "site.add", hiddenLabel: Option[String] = None)(implicit messages: Messages)
 
 @govukButton(Button(
     classes = "govuk-button--secondary",
-    content = HtmlContent(HtmlFormat.fill(
-     immutable.Seq(
-      Html(s"""<span aria-hidden="true">${messages("site.add")}</span>"""),
-      Html(s"""<span class="govuk-visually-hidden">${messages(hiddenLabel.getOrElse("site.remove"))}</span>""")
-     )
-    )),
-    name = Some("Add")
+    content = HtmlContent(HtmlFormat.fill(immutable.Seq(
+        Html(s"""<span aria-hidden="true">${messages(label)}</span>"""),
+        Html(s"""<span class="govuk-visually-hidden">${hiddenLabel.getOrElse(messages("site.add"))}</span>""")
```

Comment:
I wondered about that hiddenLabel property.   Your refactoring makes sense but unfortunately breaks other code that uses it.  That might just be on my yet-to-be-merged branch.  I'll check.

Reply:
No - this component is only used by me. I'll update my branch

Reply:
Sorry Steve. I will not merge my branch before yours. So if you want you can merge your code in and I can look into this.